client/packages/system/src/Encounter/DetailView/Toolbar.tsx

```diff
@@ -123,9 +123,7 @@ export const Toolbar: FC<ToolbarProps> = ({ encounter, onChange }) => {
                   <DatePickerInput
                     value={DateUtils.getDateOrNull(startDatetime ?? null)}
                     onChange={date => {
-                      const startDatetime = date
-                        ? DateUtils.formatRFC3339(date)
-                        : undefined;
+                      const startDatetime = DateUtils.formatRFC3339(date);
```

Comment:
No need for `date` check as this `onChange` function won't be called by the base date picker until it gets a valid date. Also, the `formatRFC3339` can handle null/undefined values anyway.